src/librustdoc/config.rs

```diff
@@ -639,6 +641,15 @@ impl Options {
         let document_hidden = matches.opt_present("document-hidden-items");
         let run_check = matches.opt_present("check");
         let generate_redirect_map = matches.opt_present("generate-redirect-map");
+        let generate_link_to_definition = matches.opt_present("generate-link-to-definition");
+
+        if generate_link_to_definition && (show_coverage || output_format != OutputFormat::Html) {
+            diag.struct_err(
+                "--generate-link-to-definition option can only be used with HTML output format",
```

Comment:
Shouldn't this mention show coverage?
```suggestion
                "--generate-link-to-definition option can only be used with HTML output format or show coverage",
```

Still looks weird, maybe we might want `with --output html or --show-coverage` to be consistent with the first part?

Reply:
Hum no, it has nothing to do with `show-coverage`, which is why we error in case it's used at the same time.

Reply:
Yes, but in the message we did not say it didn't support `show coverage`.

Reply:
Oh I see your point. `--show-coverage` doesn't output HTML, so it seemed to be excluded from my sentence. If we have to start listing all outputs, that might problematic.